schemas/session-replay/common/_slot-supported-common-record-schema.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/common/_slot-supported-common-record-schema.json",
+  "title": "SlotSupportedCommonRecordSchema",
+  "type": "object",
+  "description": "Schema of common properties for a Record event type that is supported by slots.",
+  "allOf": [
+    {
+      "$ref": "_common-record-schema.json"
+    },
+    {
+      "properties": {
+        "slotId": {
+          "type": "string",
+          "description": "Defines the unique ID of the slot that generated this record.",
```

Comment:
**🥜 nitpick:** ‏ ‏it seems used in other places but FMU all the attributes are defining something, so "defines the" seems a bit redundant to me
```suggestion
          "description": "Unique ID of the slot that generated this record.",
```